Mathlib/GroupTheory/GroupAction/MultipleTransitivity.lean

```diff
@@ -388,156 +577,137 @@ instance : IsPreprimitive (Perm α) α :=
 
 -- This is optimal, `AlternatingGroup α` is `Nat.card α - 2`-pretransitive.
 /-- A subgroup of `Perm α` is `⊤` if(f) it is `(Nat.card α - 1)`-pretransitive. -/
-theorem eq_top_if_isMultiplyPretransitive [Finite α] {G : Subgroup (Equiv.Perm α)}
```

Comment:
Maybe add a deprecation for this as well?

Reply:
The file is recent, and for the moment it seems that I am the only to have ever used it…